src/rdb.c

```diff
@@ -2113,7 +2113,7 @@ robj *rdbLoadObject(int rdbtype, rio *rdb, sds key, int dbid) {
                     return NULL;
                 }
                 streamConsumer *consumer =
-                    streamLookupConsumer(cgroup,cname,SLC_NONE,NULL);
+                    streamCreateConsumer(cgroup,cname,SCC_JUSTCREATE,NULL,0);
```

Comment:
if we have a flag for NONE, let's use it (maybe missing in other places)?
```suggestion
                    streamCreateConsumer(cgroup,cname,SCC_JUSTCREATE,NULL,SLC_NONE);
```

Reply:
The last parameter of streamCreateConsumer is dbid. Maybe I should adjust the order of the parameters.

Reply:
ohh, my bad. sorry.